addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -279,8 +279,10 @@ export class AnalyticDistribution extends Component {
                     fields: analyticAccountFields,
                     activeFields: analyticAccountFields,
                 },
-                // company domain might be required here
-                domain: [["root_plan_id", "=", account.planId]],
+                domain: [
+                    ["root_plan_id", "=", account.planId],
+                    ["company_id", "=", this.props.record.data.company_id[0]],
```

Comment:
This is a generic widget and there is nothing ensuring you this is used on a record having a `company_id` field. It's cleaner to provide the company field as a props

Reply:
Thanks!